src/themes/types.js

```diff
@@ -679,6 +679,7 @@ export type GridT = {
   margins: ResponsiveT<number>,
   gaps: ResponsiveT<number>,
   maxWidth: number,
+  unit: CSSLengthUnitT,
```

Comment:
There are a corresponding typescript theme types that should be updated too. Check `src/theme.ts`.

Reply:
Done 👍